boards/arm/gd32e507v_start/doc/index.rst

```diff
@@ -0,0 +1,142 @@
+- `GD32E50X User Manual`_
+- `GD32E507V-START User Manual`_
+
+Supported Features
+==================
+
+The board configuration supports the following hardware features:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Peripheral
+     - Kconfig option
+     - Devicetree compatible
+   * - EXTI
+     - :kconfig:option:`CONFIG_GD32_EXTI`
+     - :dtcompatible:`gd,gd32-exti`
+   * - GPIO
+     - :kconfig:option:`CONFIG_GPIO`
+     - :dtcompatible:`gd,gd32-gpio`
+   * - NVIC
+     - N/A
+     - :dtcompatible:`arm,v8m-nvic`
+   * - PWM
+     - :kconfig:option:`CONFIG_PWM`
+     - :dtcompatible:`gd,gd32-pwm`
+   * - SYSTICK
+     - N/A
+     - N/A
+   * - USART
+     - :kconfig:option:`CONFIG_SERIAL`
+     - :dtcompatible:`gd,gd32-usart`
+
+Other peripherals may be used if shields are connected to the board.
+
+Serial Port
+===========
+
+The GD32E507V-START board has one serial communication port. The default port
+is USART0 with TX connected at PB6 and RX at PB7. USART0 is exposed as a
+virtual COM port via the CN3 USB connector.
+
+Programming and Debugging
+*************************
+
+Before programming your board make sure to configure boot jumpers as
+follows:
+
+- JP3/4: Select 2-3 for both (boot from user memory)
+
+Using J-Link
+=============
+
+While the board comes with an embedded GD-Link programmer, it is not yet
+supported by OpenOCD. However, J-Link can be used to program the board using
+the SWD interface exposed in the JP1 header.
+
+#. Build the Zephyr kernel and the :ref:`hello_world` sample application:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: gd32e507v_start
+      :goals: build
+      :compact:
+
+#. Run your favorite terminal program to listen for output. On Linux the
+   terminal should be something like ``/dev/ttyUSB0``. For example:
+
+   .. code-block:: console
+
+      minicom -D /dev/ttyUSB0 -o
+
+   The -o option tells minicom not to send the modem initialization
+   string. Connection should be configured as follows:
+
+      - Speed: 115200
+      - Data: 8 bits
+      - Parity: None
+      - Stop bits: 1
+
+#. To flash an image:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: gd32e507v_start
+      :goals: flash
+      :compact:
+
+   You should see "Hello World! gd32e507v_start" in your terminal.
+
+#. To debug an image:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: gd32e507v_start
+      :goals: debug
+      :compact:
+
+.. _GigaDevice Cortex-M33 High Performance SoC Website:
+   https://www.gigadevice.com/products/microcontrollers/gd32/arm-cortex-m33/high-performance-line/
+
+.. _GD32E507X Datasheet:
+   https://gd32mcu.com/download/down/document_id/252/path_type/1
+
+.. _GD32E50X User Manual:
+   https://www.gd32mcu.com/download/down/document_id/249/path_type/1
+
+.. _GD32E507V-START User Manual:
+   https://en.maritex.com.pl/product/attachment/156365/34970e31e3ee4c41592e56875e7f7b70
```

Comment:
It's not essential, but since the link to the manual is to the distributor's copy, I think it's better to add a link to the vendor's original resource as primary information.

https://www.gd32mcu.com/data/documents/evaluationBoard/GD32E50x_Demo_Suites_V1.2.1.rar

(pdf contained in the archive.)

Reply:
good point, done